src/tests/madara_comparison.rs

```diff
@@ -12,12 +12,12 @@ fn trie_height_251() {
     let tempdir = tempfile::tempdir().unwrap();
```

Comment:
Did you tried running `cargo test --no-default-features`
I don't get how this test could run in no_std. `tempfile` require a file system which don't exist in no_std